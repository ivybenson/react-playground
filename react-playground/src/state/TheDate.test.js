import React from "react";
import ReactDOM from "react-dom";
import Messages from "./TheDate.js";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<TheDate />, div);
  ReactDOM.unmountComponentAtNode(div);
});
