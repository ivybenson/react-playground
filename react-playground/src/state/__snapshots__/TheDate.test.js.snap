// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheDate Component renders the UI as expected 1`] = `
<div>
  9/12/2020, 4:55:31 PM
</div>
`;
